### PR TITLE
Update pvc template

### PR DESCRIPTION
**Pull request by aliartiza75**
this PR will 

1. add volumeName and volumeMode in the PVC template.
2. add check for PVC name

### CODE REVIEW

**Comment by stakater-user** [comment]
@aliartiza75 Validation Successful

**Comment by aliartiza75** [comment]
@hanzala1234 please review this

**Comment by hanzala1234** [comment]
@aliartiza75  can you please make separate PR for service and pvc

**Comment by aliartiza75** [comment]
@hanzala1234 yes sure, I have also made some changes related to serviceAccount checks in deployment.

**Comment by aliartiza75** [comment]
@hanzala1234, I have created separate PR for each change.

File: application/values-test.yaml [hunk 1]
```diff
@@ -230,6 +231,8 @@ persistence:
   annotations:
   #  "helm.sh/resource-policy": keep
   storageSize: 8Gi
+  volumeMode: ""
+  volumeName: ""
```
Comment (hanzala1234) [inline application/values-test.yaml:235]: better to give some name and mode in values-test to validate these fields in pipeline

**Comment by hanzala1234** [comment]
Thanks @aliartiza75 . left small comment

**Comment by rasheedamir** [comment]
@aliartiza75 ^^ lets get them merged

**Comment by stakater-user** [comment]
@aliartiza75 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/ade671466fe910fd158325ad5730b8a17ce11918/checks) has Failed!

**Comment by stakater-user** [comment]
@aliartiza75 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/d518a104f64f6f9a271d8e39359037c9e67e8e0d/checks) has Failed!

**Comment by aliartiza75** [comment]
@rasheedamir comments are resolved.

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/d518a104f64f6f9a271d8e39359037c9e67e8e0d/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Validation Successful

**Review by hanzala1234 — APPROVED** [review]

**Pull request merged**